src/cli/java/org/commcare/util/screen/QueryScreen.java

```diff
@@ -110,7 +110,8 @@ public void init(SessionWrapper sessionWrapper) throws CommCareSessionException
         }
 
         try {
-            mTitle = Localization.get("case.search.title");
+            mTitle = Localization.get(getQueryDatum().getTitleLocaleId());
```

Comment:
This should still fall back to the old style for now, until existing apps are migrated.

Reply:
Have you confirmed this returns null if nothing is found? [This code](https://github.com/dimagi/commcare-core/blob/d9066ea561f4ce85717b25263f9ddad695c98b29/src/main/java/org/javarosa/xml/util/ActionableInvalidStructureException.java#L28-L32) makes me think it throws an exception.

Reply:
(as does the surrounding code in this file)